test/modules/paymentProcessor/PP_Streaming_v1.t.sol

```diff
@@ -358,14 +366,128 @@ contract PP_StreamingV1Test is //@note do we want to do anything about these tes
         assertEq(totalAmount, paymentClient.amountPaidCounter());
     }
 
-    // @dev Assume recipient can withdraw full amount immediately if dueTo is less than or equal to block.timestamp.
-    function testProcessPaymentsWorksForDueTimeThatIsPlacedBeforeStartTime(
+    // test cannot claim before cliff ends
+
+    function test_claimStreamedAmounts_cannotClaimBeforeCliff(
```

Comment:
That would have been tested in the existing tests, since the underlying vest calculation logic hasn't been changed. But I'll add an explicit test for that.